### PR TITLE
add:3d viewer page

### PR DESCRIPTION
**Pull request by heliprokupetz14**
Add README page for 3D catalog viewer.

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: docs/MapColonies/3D/services/3dViewer/README.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
+---
+id: 3d-model-viewer
+slug: 3d_viewer
```
Comment (syncush) [inline docs/MapColonies/3D/services/3dViewer/README.md:3]: all slugs are with `-`, why did you use `_` ? keep it uniform

File: docs/MapColonies/3D/services/3dViewer/README.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+ - Replace `<3D-CATALOG-SERVICE_URL>` with 3D CSW caralog URL.
+``` bash 
+ <3D-CATALOG-SERVICE_URL>
+ ```
+ - Replace `<token>` with with the token you have.
+``` bash 
+ <token>
+ ```
+
+Of course, you can also send a reqular request with no filter.
```
Comment (syncush) [inline docs/MapColonies/3D/services/3dViewer/README.md:64]: regular

Please add to vscode extension for spelling

File: docusaurus.config.js [hunk 3]
```diff
@@ -113,9 +113,9 @@ const config = {
             label: "Getting Started"
           },
           {to: '/blog', label: 'Release Notes', position: 'left'},
-          {to: process.env.PLAYGROUND_URL, label: 'Interactive Playground', position: 'right'},
-          {to: process.env.CATALOG_APPLICATION_URL, label: 'Catalog App', position: 'right'},
-          {to: process.env.PUZZLE_URL, label: 'Puzzle Game', position: 'right'},
+          // {to: process.env.PLAYGROUND_URL, label: 'Interactive Playground', position: 'right'},
+          // {to: process.env.CATALOG_APPLICATION_URL, label: 'Catalog App', position: 'right'},
+          // {to: process.env.PUZZLE_URL, label: 'Puzzle Game', position: 'right'},
```
Comment (syncush) [inline docusaurus.config.js:118]: why are you commenting it ?!

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,10000 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (syncush) [inline yarn.lock:1]: Please remove yarn.lock. we should use npm only.

File: package.json [hunk 5]
```diff
@@ -16,9 +16,9 @@
   },
   "dependencies": {
     "@cmfcmf/docusaurus-search-local": "^1.1.0",
-    "@docusaurus/core": "2.4.1",
-    "@docusaurus/preset-classic": "2.4.1",
-    "@docusaurus/theme-mermaid": "^2.4.1",
+    "@docusaurus/core": "^3.0.0",
+    "@docusaurus/preset-classic": "^3.0.0",
+    "@docusaurus/theme-mermaid": "^3.0.0",
```
Comment (syncush) [inline package.json:21]: when you upgraded major, did you make sure everything worked correctly?

**Review by syncush — CHANGES_REQUESTED** [review]
I will also review the guide itself.

this is just a small pr check.

File: package.json (hunk 5 above)
Comment (heliprokupetz14) [reply package.json:21]: every thing is working correctly...

File: package.json [hunk 6]
```diff
@@ -16,8 +16,8 @@
   },
   "dependencies": {
     "@cmfcmf/docusaurus-search-local": "^1.1.0",
-    "@docusaurus/core": "2.4.1",
-    "@docusaurus/preset-classic": "2.4.1",
+    "@docusaurus/core": "^2.4.1",
+    "@docusaurus/preset-classic": "^2.4.1",
```
Comment (syncush) [inline package.json:20]: look at the changes from previous version of the code, don't use `^` , we are sticking to a specific version of docusarus

**Review by syncush — CHANGES_REQUESTED** [review]
please take a new screenshot without the bookmarks.

**Review by syncush — APPROVED** [review]

**Review by syncush — APPROVED** [review]

**Pull request merged**